grammars/shell-unix-bash.cson

```diff
@@ -119,13 +119,21 @@
       }
     ]
   'comment':
-    'begin': '(^[ \\t]+)?(?<!\\S)(?=#)(?!#\\{)'
+    'begin': '(^[ \\s]+)?(?<!\\S)(?=#)(?!#\\{)'
```

Comment:
I think this could just be `\\s+` now instead of `[ \\s]+` since space is covered by `\s`